examples/README.md

```diff
@@ -14,14 +14,45 @@ and streaming APIs.
 You can find examples that demonstrate these APIs
 in the [`examples/src/`](./src) directory.
 
+* [How to encrypt and decrypt](./src/onestep_defaults.py)
+* [How to change the algorithm suite](./src/onestep_unsigned.py)
+* [How to encrypt and decrypt data streams in memory](./src/in_memory_streaming_defaults.py)
+* [How to encrypt and decrypt data streamed between files](./src/file_streaming_defaults.py)
+
 ## Configuration
 
 To use the library APIs,
 you need to describe how you want the library to protect your data keys.
 You can do this using
```

Comment:
```suggestion
You can do this by instantiating/configuring/creating ...
```
Choose the best verb for this idiom, but it needs to be prefixed with "by."

Reply:
```suggestion
You can do this by configuring
```